styles.css

```diff
@@ -0,0 +1,131 @@
+body {
+  position: relative;
+  width: 100%;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  text-align: start;
+}
+
+.title_lo {
+  margin-top: 50px;
+  margin-left: 50px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.logo {
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.logo:hover {
+  color: blueviolet;
+}
+
+.fas {
+  width: 200px;
+  font-size: 20px;
+  margin-left: 200px;
+}
+
+.fas:hover {
+  color: #6070ff;
+}
+
+.content-name {
+  background-image: url('./src-image/header-shapes-mobile@2x.svg');
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.my_name {
+  width: 310px;
+  padding-left: 24px;
+  padding-top: 62px;
+
+  /* Header / 2 */
```

Comment:
[OPTIONAL]
- [ ]  Please delete any unnecessary comments, they can make your project look messy. A well-structured code is clear by itself. :sunglasses: